### PR TITLE
Bears: Use only important bears for each language

### PR DESCRIPTION
**Pull request by hypothesist**

### CODE REVIEW

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala-quickstart/pull/26/commits/4a8819e1e70a1e99aeef4ab910bf2201033e2637 that was long overdue

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala-quickstart/pull/26/commits/54f21c6b9bf67e4fb0c6be33cfb4398d69650338

File: coala_quickstart/generation/Bears.py [hunk 1]
```diff
@@ -24,9 +24,29 @@ def filter_relevant_bears(used_languages):
...
+                       "ClangComplexityBear",
+                       "CSecurityBear",
+                       "GNUIndentBear"],
+                 "JavaScript": ["JSHintBear",
+                                "JSComplexityBear"],
+                 "Java": ["JavaPMD",
+                          "CheckstyleBear"],
+                 "Python": ["PEP8Bear",
+                            "PyLintBear",
+                            "YapfBear"]}
```
Comment (sils1297) [inline coala_quickstart/generation/Bears.py]: make them sets instead of lists and move this into Constants with an upper case variable name?

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala-quickstart/pull/26/commits/6a7b63a61804fca774c73d20316138e92a6451f7

**Comment by hypothesist** [comment]
@rultor merge

**Pull request merged**